app/data_models/list_store.py

```diff
@@ -60,11 +61,11 @@ def serialize(self):
 
         return serialized
 
-    def __repr__(self):
+    def __repr__(self) -> str:
         return f"<ListModel name={self.name} items={self.items}, primary_person={self.primary_person}>"
 
     @property
-    def first(self):
+    def first(self) -> Any:
```

Comment:
This returns a string.

Reply:
Same as above, I think it confuses standard library `builtins` `items` where the type is not define with our var which has the same name, so it becomes `Any`.